packages/cactus-test-tooling/src/main/typescript/iroha/iroha-test-ledger.ts

```diff
@@ -0,0 +1,352 @@
+    const fnTag = "IrohaTestLedger#getContainer()";
+    if (!this.container) {
+      throw new Error(`${fnTag} container not yet started by this instance.`);
+    } else {
+      return this.container;
+    }
+  }
+
+  public get imageFqn(): string {
+    return `${this.imageName}:${this.imageVersion}`;
+  }
+
+  public async getRpcToriiPortHost(): Promise<string> {
+    const ipAddress = "127.0.0.1";
+    const hostPort: number = await this.getRpcToriiPort();
+    return `http://${ipAddress}:${hostPort}`;
+  }
+
+  /**
+   * Output is based on the standard Iroha genesis.block contents.
+   *
+   * @see https://github.com/hyperledger/iroha/blob/main/example/genesis.block
+   */
+  public getGenesisAdminAccount(): string {
+    return "admin@test";
+  }
+
+  /**
+   * Output is based on the standard Iroha admin user public key.
+   *
+   * @see https://github.com/hyperledger/iroha/blob/main/example/admin%40test.pub
+   * @see https://github.com/hyperledger/iroha/blob/main/example/genesis.block
+   */
+  public getGenesisAccountPubKey(): string {
+    return "313a07e6384776ed95447710d15e59148473ccfc052a681317a72a69f2a49910";
+  }
+
+  /**
+   * Output is based on the standard Iroha admin user private key.
+   *
+   * @see https://github.com/hyperledger/iroha/blob/main/example/admin%40test.priv
+   */
+  public getGenesisAccountPrivKey(): string {
+    return "f101537e319568c765b2cc89698325604991dca57b9716b58016b253506cab70";
+  }
+
+  public async getNodeKeyPair(): Promise<IKeyPair> {
+    const fnTag = `IrohaTestLedger#getNodeKeyPair()`;
+    if (!this.container) {
+      throw new Error(`${fnTag} this.container cannot be falsy.`);
+    }
+    const publicKey = await Containers.pullFile(
+      this.container,
+      "/opt/iroha_data/node0.pub",
+    );
+    const privateKey = await Containers.pullFile(
+      this.container,
+      "/opt/iroha_data/node0.priv",
+    );
+    return { publicKey, privateKey };
+  }
+
+  public async start(omitPull = false): Promise<Container> {
+    if (this.container) {
+      await this.container.stop();
+      await this.container.remove();
+    }
+    const docker = new Docker();
+    if (!omitPull) {
+      this.log.debug(`Pulling container image ${this.imageFqn} ...`);
+      await Containers.pullImage(this.imageFqn);
+      this.log.debug(`Pulled ${this.imageFqn} OK. Starting container...`);
+    }
+
+    return new Promise<Container>((resolve, reject) => {
+      const eventEmitter: EventEmitter = docker.run(
+        this.imageFqn,
+        [],
+        [],
+        {
+          ExposedPorts: {
+            [`${this.rpcToriiPort}/tcp`]: {}, // Iroha RPC - Torii
+          },
+          Env: this.envVars,
+          Healthcheck: {
+            Test: ["CMD-SHELL", "python3 /healthcheck.py admin@test 50051"],
+            Interval: 1000000000, // 1 second
+            Timeout: 3000000000, // 3 seconds
+            Retries: 299,
+            StartPeriod: 3000000000, // 1 second
```

Comment:
I suggest constant for 1 second - comments would be not necessarily.

BTW. I think here is not correct comment:
```
            StartPeriod: 3000000000, // 1 second
```

Reply:
Thank you! Done!